Source/svg/SVGParser.swift

```diff
@@ -1187,6 +1266,17 @@ open class SVGParser {
         }
         return .none
     }
+    
+    fileprivate func parseIdFromUrl(_ urlString: String) -> String? {
+        if urlString.hasPrefix("url") {
+            let index = urlString.index(urlString.startIndex, offsetBy: 4)
+            return String(urlString.suffix(from: index))
+                .replacingOccurrences(of: "(", with: "")
```

Comment:
It's not quite efficient. Better to make a substring once instead of creating 4 intermediate strings.